tests/rkt_tests.go

```diff
@@ -317,3 +360,201 @@ func checkAppStatus(t *testing.T, ctx *testutils.RktRunCtx, multiApps bool, appN
 			appName, expected, err)
 	}
 }
+
+type imageInfo struct {
+	id         string
+	name       string
+	version    string
+	importTime int64
+	manifest   []byte
+}
+
+type appInfo struct {
+	name     string
+	exitCode int
+	image    *imageInfo
+	// TODO(yifan): Add app state.
+}
+
+type networkInfo struct {
+	name string
+	ipv4 string
+}
+
+type podInfo struct {
+	id       string
+	pid      int
+	state    string
+	apps     map[string]*appInfo
+	networks map[string]*networkInfo
+	manifest []byte
+}
+
+// parsePodInfo parses the 'rkt status $UUID' result into podInfo struct.
+// For example, the 'result' can be:
+// state=running
+// networks=default:ip4=172.16.28.103
+// pid=14352
+// exited=false
+func parsePodInfoOutput(t *testing.T, result string, p *podInfo) {
+	lines := strings.Split(strings.TrimSuffix(result, "\n"), "\n")
+	for _, line := range lines {
+		tuples := strings.SplitN(line, "=", 2)
+		if len(tuples) != 2 {
+			t.Fatalf("Unexpected line: %v", line)
+		}
+
+		switch tuples[0] {
+		case "state":
+			p.state = tuples[1]
+		case "networks":
+			networks := strings.Split(tuples[1], ",")
+			for _, n := range networks {
+				fields := strings.Split(n, ":")
+				if len(fields) != 2 {
+					t.Fatalf("Unexpected network info format: %v", n)
+				}
+
+				ip4 := strings.Split(fields[1], "=")
+				if len(ip4) != 2 {
+					t.Fatalf("Unexpected network info format: %v", n)
+				}
+
+				networkName := fields[0]
+				p.networks[networkName] = &networkInfo{
+					name: networkName,
+					ipv4: ip4[1],
+				}
+			}
+		case "pid":
+			pid, err := strconv.Atoi(tuples[1])
+			if err != nil {
+				t.Fatalf("Cannot parse the pod's pid %q: %v", tuples[1], err)
+			}
+			p.pid = pid
+		}
+		if strings.HasPrefix(tuples[0], "app-") {
+			exitCode, err := strconv.Atoi(tuples[1])
+			if err != nil {
+				t.Fatalf("cannot parse exit code from %q : %v", tuples[1], err)
+			}
+			appName := strings.TrimPrefix(tuples[0], "app-")
+
+			for _, app := range p.apps {
+				if app.name == appName {
+					app.exitCode = exitCode
+					break
+				}
+			}
+		}
+	}
+}
+
+func getPodDir(t *testing.T, ctx *testutils.RktRunCtx, podID string) string {
+	podsDir := path.Join(ctx.DataDir(), "pods")
+
+	dirs, err := ioutil.ReadDir(podsDir)
+	if err != nil {
+		t.Fatalf("Unexpected error: %v", err)
+	}
+
+	for _, dir := range dirs {
+		podDir := path.Join(podsDir, dir.Name(), podID)
+		if _, err := os.Stat(podDir); err == nil {
+			return podDir
+		}
+	}
+	t.Fatalf("Failed to find pod directory for pod %q", podID)
+	return ""
+}
+
+// getPodInfo returns the pod info for the given pod ID.
+func getPodInfo(t *testing.T, ctx *testutils.RktRunCtx, podID string) *podInfo {
+	p := &podInfo{
+		id:       podID,
+		apps:     make(map[string]*appInfo),
+		networks: make(map[string]*networkInfo),
+	}
+
+	// Read pod manifest.
+	podManifestPath := path.Join(getPodDir(t, ctx, podID), "pod")
+	f, err := os.Open(podManifestPath)
```

Comment:
I'd like if there was a `rkt` command to get the manifest without doing all of this manually. Then, `getPodDir()` would not be needed.

For now, can you add a comment:
```
// TODO: run "rkt cat-manifest" instead of opening the manifest file manually.
// https://github.com/coreos/rkt/issues/1730
```